tests/search/e2e/integration/wp-cli.spec.js

```diff
@@ -176,34 +176,10 @@ describe('WP-CLI Commands', () => {
 		});
 	});
 
-	it('Can set the algorithm version', () => {
-		cy.wpCli('wp vip-search set-algorithm-version --default')
-			.its('stdout')
-			.should('contain', 'Done');
-
-		cy.wpCli('wp vip-search get-algorithm-version')
-			.its('stdout')
-			.should('contain', 'default');
-
-		cy.wpCli('wp vip-search set-algorithm-version --version=1.0.0')
-			.its('stdout')
-			.should('contain', 'Done');
-
-		cy.wpCli('wp vip-search get-algorithm-version').its('stdout').should('contain', '1.0.0');
-
-		cy.wpCli('wp vip-search set-algorithm-version', true)
-			.its('stderr')
-			.should('contain', 'This command expects a version number or the --default flag');
-	});
```

Comment:
Why remove these tests?

Reply:
Because we've disabled the `set-algorithm-version` command. If the algorithm version needs to be set, it should be done by filter.